sda-commons-server-morphia/README.md

```diff
@@ -103,6 +103,10 @@ _Please note the double quotes around the values.
  Without them for `caCertificate`, Dropwizard will not be able to load the certificate correctly because the `MONGODB_CA_CERTIFICATE` variable contains line breaks.  
  Omitting the double quotes for other values like `password` can cause trouble in case of certain special characters (e.g. colon)_
 
+ _Instead of setting `caCertificate` in the environment, we should use the default configuration of the
+ [`CaCertificatesBundle`](../sda-commons-shared-certificates/src/main/java/org/sdase/commons/shared/certificates/ca/CaCertificatesBundle.java),
+ and mount CA Certificates to the default directory `DEFAULT_TRUSTED_CERTIFICATES_DIR = "/var/trust/certificates"`_
+
```

Comment:
```suggestion
#### CA Certificates support

Instead of providing `caCertificate` as an environment variable, we can add Ca certificates PEM files in the directory `/var/trust/certificates`. Certificates available in sub-directories will also be loaded.

Note that this directory is also configurable through the Dropwizard config class. The config class should then provide a 
[`CaCertificateConfiguration`](../sda-commons-shared-certificates/src/main/java/org/sdase/commons/shared/certificates/ca/CaCertificateConfiguration.java) to the bundle builder.
See [`sda-commons-shared-certificates`](../sda-commons-shared-certificates/README.md) for details.

```
I think, if we are still preferring  the old way other the new one (if the are both used) then we should just present it as an option in morphia bundle documentation, and if we want to deprecate it in an other release (change the order of preference), at least then we would be sure that it is stable after at least testing that in one service.

Reply:
Deprecation is not the same as preference. The deprecation is only an announcement that an API will be removed later. Until removal (or at least until the next major release) everything should behave the same as before so that users are not forced to change anything. So preferring the old way over the new way complies with the deprecation in my opinion. Even when we are suggesting to change configuration to use the new way.

Reply:
I get your point now

Reply:
We should also remember to announce the deprecation in the release notes after merge by editing them in the GitHub UI. Unfortunately, semantic release is not able to handle deprecations from commit messages. But I also prefer that we keep one commit that includes information about the deprecation in the final PR to master. But we will update the commit history in that PR anyways.

Reply:
Sure, will keep in mind!